src/Entity/Sylius/Order.php

```diff
@@ -349,6 +356,43 @@
  *      "path"="/orders/{id}/players",
  *      "controller"=AddPlayer::class,
  *     },
+ *     "loopeat_formats"={
+ *       "method"="GET",
+ *       "path"="/orders/{id}/loopeat_formats",
+ *       "controller"=LoopeatFormatsController::class,
+ *       "output"=LoopeatFormatsOutput::class,
+ *       "normalization_context"={"api_sub_level"=true},
+ *       "security"="is_granted('view', object)",
+ *       "openapi_context"={
+ *         "summary"="Get Loopeat formats for an order"
+ *       }
+ *     },
+ *     "update_loopeat_formats"={
+ *       "method"="PUT",
+ *       "path"="/orders/{id}/loopeat_formats",
+ *       "controller"=UpdateLoopeatFormatsController::class,
+ *       "security"="is_granted('view', object)",
+ *       "input"=LoopeatFormatsOutput::class,
+ *       "validate"=false,
+ *       "normalization_context"={"groups"={"cart"}},
+ *       "denormalization_context"={"groups"={"update_loopeat_formats"}},
+ *       "openapi_context"={
+ *         "summary"="Update Loopeat formats for an order"
+ *       }
+ *     },
+ *     "update_loopeat_returns"={
+ *       "method"="POST",
```

Comment:
Should it be a "PUT" method like `update_loopeat_formats` ?

Reply:
Oops, the changes on the app have been merged & deployed, and it using POST

https://github.com/coopcycle/coopcycle-app/blob/6675e8a18177ca64bfe5fe79216087cf2358ad88/src/redux/Checkout/actions.js#L1360